skeleton/aws/main.tf

```diff
@@ -1,24 +1,3 @@
-terraform {
-  cloud {
-    organization = "organization"
-
-    workspaces {
-      name = "terraform_workspace"
-    }
-  }
-}
```

Comment:
This part seems to belong to the main, more than the provider part, no? If not, can you share more insights about what makes it belong to "providers" (I'm still discovering Terraform 😇 )
Thanks! :)

Reply:
We don't have any convention for this, but as I just want to move the provider into a separate file, this block also has `required_providers` attribute, so I moved it as well.
Then later, when adding a new provider, we just need to modify that file only. What do you think?
[The standard files](https://www.terraform.io/language/modules/develop/structure) don't have `providers.tf` file :D

Reply:
 It seems like there can be multiple `terraform` blocks... if yes, should we make it like this? (to be tested :P ) 
 
 
<img width="466" alt="image" src="https://user-images.githubusercontent.com/77609814/179701378-2f4f0e13-ef68-498d-b6bc-d0bc750a73f6.png">

Reply:
If not, let's go for the "all in providers" solution as you initially implemented ^^,

Reply:
Good catch, we can use multiple terraform blocks 🙌 
I fixed in https://github.com/nimblehq/infrastructure-templates/pull/69/commits/61af80a5a8843e83abf9bd6d0c3c938fb3a3971d